### PR TITLE
Update Querit results

### PR DESCRIPTION
**Pull request by moshesbeta**
### Checklist

- [x] My model has a model sheet, report, or similar
- [x] My model has a reference implementation in [`mteb/models/model_implementations/`](https://github.com/embeddings-benchmark/mteb/tree/main/mteb/models/model_implementations), this can be as an API. Instruction on how to add a model can be found [here](https://embeddings-benchmark.github.io/mteb/contributing/adding_a_model/) 
  - [ ] No, but there is an existing PR https://github.com/embeddings-benchmark/mteb/pull/4569
- [x] The results submitted are obtained using the reference implementation
- [x] My model is available, either as a publicly accessible API or publicly on e.g., Huggingface
- [x] I *solemnly swear* that for all results submitted I have not trained on the evaluation dataset including training splits. If I have, I have disclosed it clearly.

### CODE REVIEW

File: results/Querit__Querit/b611630803ed857ecb159ea9df75738bdd40d1c0/AskUbuntuDupQuestions.json
Comment (KennethEnevoldsen) [inline results/Querit__Querit/b611630803ed857ecb159ea9df75738bdd40d1c0/AskUbuntuDupQuestions.json:1]: this is deleted but never added

File: results/Querit__Querit/e50d4d8183ccec6923ac2b256c755ddd3ed24066/AlloprofReranking.json [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        "nauc_mrr_at_1000_std": 0.194945,
+        "nauc_mrr_at_1000_diff1": 0.556199,
+        "hit_rate_at_1": 0.69603,
+        "hit_rate_at_3": 0.89162,
+        "hit_rate_at_5": 0.94301,
+        "hit_rate_at_10": 0.98575,
+        "hit_rate_at_20": 1.0,
+        "hit_rate_at_100": 1.0,
+        "hit_rate_at_1000": 1.0,
+        "main_score": 0.79194,
```
Comment (KennethEnevoldsen) [inline results/Querit__Querit/e50d4d8183ccec6923ac2b256c755ddd3ed24066/AlloprofReranking.json]: Seems like some of the scores have changed, can you just describe what have lead to this?

Comment (moshesbeta) [reply results/Querit__Querit/e50d4d8183ccec6923ac2b256c755ddd3ed24066/AlloprofReranking.json]: We further refined the training data and integrated the weights for different checkpoints to improve performance.

Comment (KennethEnevoldsen) [reply results/Querit__Querit/e50d4d8183ccec6923ac2b256c755ddd3ed24066/AlloprofReranking.json]: Perfect, then you need to change the reference revision in MTEB: https://github.com/embeddings-benchmark/mteb/blob/841b67098c5ef2418726e918207a290f57bdd35d/mteb/models/model_implementations/querit_models.py#L229-L230

And rerun all the submitted results (see comment above)

Comment (moshesbeta) [reply results/Querit__Querit/e50d4d8183ccec6923ac2b256c755ddd3ed24066/AlloprofReranking.json]: Thank you for your reply. We have submitted our pull request here and are awaiting review. We will keep you updated on the review results.

Comment (Samoed) [reply results/Querit__Querit/e50d4d8183ccec6923ac2b256c755ddd3ed24066/AlloprofReranking.json]: @KennethEnevoldsen https://github.com/embeddings-benchmark/mteb/pull/4569

File: results/Querit__Querit/b611630803ed857ecb159ea9df75738bdd40d1c0/AskUbuntuDupQuestions.json
Comment (KennethEnevoldsen) [reply results/Querit__Querit/b611630803ed857ecb159ea9df75738bdd40d1c0/AskUbuntuDupQuestions.json:1]: this is still missing

Comment (moshesbeta) [reply results/Querit__Querit/b611630803ed857ecb159ea9df75738bdd40d1c0/AskUbuntuDupQuestions.json:1]: Our model primarily focuses on multilingual reranking tasks, and it's not tested on this dataset. Is it necessary to provide results from this test set?
